RELEASE/scripts/autoscend/auto_acquire.ash

```diff
@@ -1010,7 +1006,6 @@ boolean LX_craftAcquireItems()
 		}
 	}
 
-	if(!in_community())
 	{
```

Comment:
extra scope still left here which is just cosmetic. Up to you if you want to remove it or not.